src/test/ui/feature-gate-isa_attribute.rs

```diff
@@ -0,0 +1,5 @@
+#[instruction_set]
+//~^ ERROR the `#[instruction_set]` attribute is an experimental feature
+//~| ERROR E0658, E0778
```

Comment:
You need to list all errors that will be output.
```suggestion
//~| ERROR malformed `instruction_set` attribute input
//~| ERROR must specify an instruction set [E0778]
```